components/ui/menu/SidebarMenu.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <nav class="sidebar-menu">
+    <h2 class="sidebar-menu__title">
+      Elements
+    </h2>
+    <MenuLink
+      v-for="link in qiskitElements"
+      :key="link.url"
+      :class="['sidebar-menu__link', isActive(link) && 'sidebar-menu__link_active']"
```

Comment:
Non-blocking: I think we should use the pattern:
```
class="sidebar-menu__link"
:class="isActive(link) && 'sidebar-menu__link_active'"
```
Or even better:
```
class="sidebar-menu__link"
:class="{'sidebar-menu__link_active': isActive(link)}"
```

To make a clear separation between dynamic and non-dynamic classes.